x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_edit/pipelines_edit.tsx

```diff
@@ -95,25 +95,20 @@ export const PipelinesEdit: React.FunctionComponent<RouteComponentProps<MatchPar
 
   if (isLoading) {
     return (
-      <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
+      <EuiPageSection alignment="center" color="subdued">
         <SectionLoading>
           <FormattedMessage
             id="xpack.ingestPipelines.edit.loadingPipelinesDescription"
             defaultMessage="Loading pipeline…"
           />
         </SectionLoading>
-      </EuiPageContent>
+      </EuiPageSection>
     );
   }
 
   if (error) {
     return (
-      <EuiPageContent
-        verticalPosition="center"
-        horizontalPosition="center"
-        color="danger"
-        data-test-subj="fetchPipelineError"
-      >
+      <EuiPageSection alignment="center" color="danger" data-test-subj="fetchPipelineError">
```

Comment:
`EuiPageTemplate.EmptyPrompt` should be used here

```suggestion
      <EuiPageTemplate.EmptyPrompt
      iconType="warning"
      color="danger"
      title={
        <h2>
          <FormattedMessage
            id="xpack.ingestPipelines.edit.fetchPipelineError"
            defaultMessage="Unable to load '{name}'"
            values={{ name: decodedPipelineName }}
          />
        </h2>
      }
      body={<p>Error</p>}
      actions={
        <EuiButton onClick={resendRequest} iconType="refresh" color="danger">
          <FormattedMessage
            id="xpack.ingestPipelines.edit.fetchPipelineReloadButton"
            defaultMessage="Try again"
          />
        </EuiButton>
      }
    />
```

// Curent
<img width="969" alt="Screenshot 2023-08-04 at 12 15 24 PM" src="https://github.com/elastic/kibana/assets/5226211/83765b09-544c-402f-a2e4-594ea9c18d40">


// Expected
<img width="624" alt="Screenshot 2023-08-04 at 12 14 10 PM" src="https://github.com/elastic/kibana/assets/5226211/4b35dc05-97f7-4866-8175-9cf0d95fec35">